src/BinDeps.jl

```diff
@@ -523,8 +524,14 @@ function run(s::SynchronousStepCollection)
     end
 end
 
-is_unix() && (make_command = `make -j8`)
-is_windows() && (make_command = `make`)
+const make_command =
```

Comment:
and... please also check this: https://github.com/JuliaLang/BinDeps.jl/blob/25c17a227488eb231d9b8bccfee471b469d9c572/src/BinDeps.jl#L390-L398